src/vmecpp/__init__.py

```diff
@@ -211,13 +211,28 @@ def _validate_fourier_coefficients_shapes(self) -> typing.Self:
         for field in mpol_two_ntor_plus_one_fields:
             shape = np.shape(getattr(self, field))
             if shape != expected_shape:
-                msg = (
-                    f"{field} has shape {shape} instead of the expected {expected_shape}. "
-                    f"Does your coefficient array exceed mpol={self.mpol} or ntor={self.ntor}?"
-                )
+                msg = f"{field} has shape {shape} instead of the expected {expected_shape}\n"
+                f"Does your coefficient array exceed mpol={self.mpol} or ntor={self.ntor}?"
                 raise ValueError(msg)
         return self
 
+    @pydantic.model_validator(mode="after")
+    def _validate_stellarator_stellarator_asymmetric_fields(self) -> typing.Self:
+        """Check if all fields that break stellarator symmetry match the lasym flag."""
+        ASYMMETRIC_FIELDS = ["rbs", "zbc", "zaxis_c", "raxis_s"]
+        # If stellarator-symmetric
+        if not self.lasym:
```

Comment:
optional: why use a comment when we can use code 😬 
```suggestion
        is_stellarator_symmetric = not self.lasym
        if is_stellarator_symmetric:
```